packages/nextjs/components/Header.tsx

```diff
@@ -39,6 +39,9 @@ export const Header = () => {
       <li>
         <NavLink href="/">Home</NavLink>
       </li>
+      <li>
+        <NavLink href="/dice">Dice Game</NavLink>
```

Comment:
Please add Icon from `heroicons` here. This ones are good I think
<img width="413" alt="image" src="https://github.com/scaffold-eth/se-2-challenges/assets/25638585/d4360312-e4f8-44f4-ae16-288c6de86d44">

Reply:
Thanks was looking for this icon